.meshrc.yaml

```diff
@@ -104,17 +104,11 @@ sources:
             headers:
               Cookie: "{context.headers['cookie']}"
           - type: Query
-            field: Samples
-            path: /pub/{args.snapshotLinkId}/samples/{args.sampleId}.json
```

Comment:
Just checking - does this still work if we need to include the snapshotLinkId?

Reply:
I don't think this GraphQL endpoint was even being used. `snapshotLinkId` is only used for short read MNGS, so I didn't port that over to the new endpoint.